### PR TITLE
Update CODEOWNERS

### LINKED ISSUE #578: Promote GaboFDC to Write
Welcome to the fold @GaboFDC !
You now have write...

### PR DESCRIPTION
**Pull request by admiralAwkbar**
This closes #578

### CODE REVIEW

**Comment by admiralAwkbar** [comment]
@GaboFDC

**Comment by admiralAwkbar** [comment]
@GaboFDC  also...

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ######################################################################
 # These owners will be the default owners for everything in the repo #
 ######################################################################
-*       @admiralawkbar @jwiebalk @zkoppert @IAmHughes @nemchik @Hanse00 @dependabot-bot
+*       @admiralawkbar @jwiebalk @zkoppert @IAmHughes @nemchik @Hanse00 @dependabot-bot @GaboFDC
```
Comment (nhutran1201) [inline .github/CODEOWNERS:4]: 931eba8

**Comment by GaboFDC** [comment]
Thnaks all

**Pull request merged**